frontend/src/config/mongolia/prism.json

```diff
@@ -1,5 +1,6 @@
 {
   "country": "Mongolia",
+  "countryAdmin0Id": 44,
```

Comment:
let's add the real number here,  `167`

Reply:
removed this as we're not including charts in mongolia